docs/dg/dev/backend-development/audit-logs/audit-logs.md

```diff
@@ -0,0 +1,265 @@
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_ZED] = [
+    ZedSecurityAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE] = [
+    GlueSecurityAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_GLUE_BACKEND] = [
+    GlueBackendSecurityAuditLoggerConfigPlugin::class,
+];
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_MERCHANT_PORTAL] = [
+    MerchantPortalSecurityAuditLoggerConfigPlugin::class,
+];
+```
+
+## Default plugins
+
+By default, there are plugins for the `security` log type. These plugins are preconfigured and used to log security-related events in various applications.
+
+## Adding audit logs
+
+Audit logs are added using the `AuditLoggerTrait` trait. Example:
+
+```php
+<?php
+
+use Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer;
+use Spryker\Shared\Log\AuditLoggerTrait;
+
+class AuditLogger
+{
+    use AuditLoggerTrait;
+
+    /**
+     * @param string $action
+     * @param list<string> $tags
+     *
+     * @return void
+     */
+    public function addAuditLog(string $action, array $tags): void
+    {
+        $this->getAuditLogger(
+            (new AuditLoggerConfigCriteriaTransfer())->setChannelName('security'),
+        )->info('user logged in', ['tags' => ['user_logged_in']]);
+    }
+}
+```
+
+When adding audit logs, we recommend including tags. Tags provide additional context and help categorize logs, making them easier to search and analyze. In the prior example, the tag `user_logged_in` is used to indicate a user login action.
+
+We don't recommend tracking all database related activity actions in the audit logs, such as user status updates or customer email updates. Logging every database interaction can quickly lead to excessive log volume, making it harder to find relevant information and potentially impacting system performance. Focus on logging critical actions that have significant security implications.
+
+## Example of audit log data
+
+This example of an audit log entry was recorded during a successful login attempt:
+
+
+<details>
+  <summary>Audit log example</summary>
+```json
+{
+    "@timestamp": "2024-07-16T09:45:12.310532+00:00",
+    "@version": 1,
+    "host": "b0bc3b05abf2",
+    "message": "Successful Login",
+    "type": "YVES",
+    "channel": "security",
+    "level": "INFO",
+    "monolog_level": 200,
+    "extra": {
+        "environment": {
+            "application": "YVES",
+            "environment": "docker.dev",
+            "store": null,
+            "codeBucket": "EU",
+            "locale": "en_US"
+        },
+        "server": {
+            "url": "http://yves.de.spryker.local/en/login_check",
+            "is_https": false,
+            "hostname": "yves.de.spryker.local",
+            "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36",
+            "user_ip": "192.168.000.00",
```

Comment:
why is it incorrect? do we apply some hidden transformations?

Reply:
```suggestion
            "user_ip": "192.168.0.1",
```

Reply:
fixed